### PR TITLE
rg-206: Add Header To Template Editor

### PR DESCRIPTION
**Pull request by Valentyn13**

### CODE REVIEW

**Comment by Valentyn13** [comment]
[The header is missing on the "Template editor" page.](https://trello.com/c/lP2anwBX/206-the-header-is-missing-on-the-template-editor-page)

**Review by siefimov — APPROVED** [review]

**Review by honkster-nazarii — APPROVED** [review]

File: frontend/src/bundles/edit-template/edit-template.tsx [hunk 1]
```diff
@@ -62,49 +72,55 @@ const EditTemplatePage: React.FC = () => {
...
+                    className={clsx(
+                        editorStyles.editor_sidebar__nav,
+                        templateStyles.editor_sidebar__nav,
+                    )}
+                >
+                    <ul className={editorStyles.editor_sidebar__list}>
+                        {templateBlockTitles.map((block) => (
+                            <li
+                                key={block}
+                                style={{ 'display': 'flex' }}
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/edit-template/edit-template.tsx:96]: You can add it to class

Comment (Valentyn13) [reply frontend/src/bundles/edit-template/edit-template.tsx]: done

**Review by Vitaliy-Kalenichenko — CHANGES_REQUESTED** [review]
https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/pull/217/files#r1525249028

**Pull request closed without merging**